Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Projectiles/railgun_ammo.yml

```diff
@@ -0,0 +1,88 @@
+- type: entity
+  parent: BaseItem
+  id: BulletRailgun
+  abstract: true
+  components:
+  - type: Item
+    size: Small
+  - type: Sprite
+    sprite: DeltaV/Objects/Weapons/Guns/Ammunition/Cartridges/railrod.rsi
+    layers:
+      - state: base
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape: !type:PhysShapeCircle
+          radius: 0.2
+        density: 10
+        restitution: 0.3
+        friction: 0.2
+        mask:
+        - ItemMask
+      projectile:
+        shape:
+          !type:PhysShapeAabb
+          bounds: "-0.1,-0.1,0.1,0.1"
+        hard: false
+        mask:
+        - Impassable
+        - BulletImpassable
+  - type: EmbeddableProjectile
+    minimumSpeed: 10
+    sound: /Audio/DeltaV/Weapons/Guns/Impacts/rod_impact.ogg
+    embedOnThrow: false
+  - type: ThrowingAngle
+    angle: 0
+  - type: Ammo
+    muzzleFlash: null
+  - type: Tag
+    tags:
+    - BulletRailgun
```

Comment:
You haven't defined this tag anywhere;

should be done in prototypes/delta-v/tags.yml